client/state/rewind/test/selectors.js

```diff
@@ -107,3 +109,30 @@ describe( 'getBackupRetentionDays()', () => {
 		}
 	);
 } );
+describe( 'getBackupRetentionUpdateRequestStatus()', () => {
+	const TEST_SITE_ID = 123;
+
+	test( 'should default to UNSUBMITTED when no backup retention update request is being made yet.', () => {
+		const state = {
+			rewind: {
+				[ TEST_SITE_ID ]: {},
+			},
+		};
+		expect( getBackupRetentionUpdateRequestStatus( state, TEST_SITE_ID ) ).toEqual(
+			BACKUP_RETENTION_UPDATE_REQUEST.UNSUBMITTED
+		);
+	} );
```

Comment:
Worth having a test with a site that doesn't exist in `state.rewind` at all.

Reply:
Added :)